docs/settings/security-settings.asciidoc

```diff
@@ -162,6 +162,51 @@ In addition to <<authentication-provider-settings,the settings that are valid fo
 
 |===
 
+[float]
+[[anonymous-authentication-provider-settings]]
+===== Anonymous authentication provider settings
+
+In addition to <<authentication-provider-settings,the settings that are valid for all providers>>, you can specify the following settings:
+
+[NOTE]
+============
+You can configure only one anonymous provider per {kib} instance.
+============
+
+[cols="2*<"]
+|===
+| `xpack.security.authc.providers.`
+`anonymous.<provider-name>.credentials` {ess-icon}
+| Credentials that {kib} should use internally to authenticate anonymous requests to {es}. Possible values are: username and password credentials, API key or `elasticsearch_anonymous_user` constant if you want to leverage {ref}/anonymous-access.html[{es} anonymous access].
+
+2+a| For example:
+
+[source,yaml]
+----------------------------------------
+# Username and password credentials
```

Comment:
__note:__ not super happy about having technically invalid yaml snippet (because of duplicated entries), but couldn't think of a better option.

Reply:
I think this is reasonable. We could split this out into 4 different snippets if necessary, but I think what you have here is fine.